.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  release:
+    types: [published]
+
+name: release
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Get release
+      id: get_release
+      uses: bruceadams/get-release@v1.2.2
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
+
+    - name: Checkout repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15.3
```

Comment:
For this, I think "at least 1.15" is sufficient; i.e. `^1.15`.  If .3 point release is required to build then what you have is appropriate.  If you want it locked to EXACTLY 1.15.3, it'd be `1.15.3`

Reply:
Agreed! https://github.com/StackExchange/dnscontrol/commit/bf7cb5bb03117a5d879383f4eab29564dc8c0f1a